audit.py

```diff
@@ -36,16 +44,104 @@ def reverse_numeric_sort_order(number: int):
 # token=TOKEN -- required
 
 
-class OutputFormat(str, Enum):
-    text = "text"
-    json = "json"
-    html = "html"
+class OutputFormat(StrEnum):
+    """
+    The output format fro reports
+    """
+    text = auto()
+    json = auto()
+    html = auto()
 
     @staticmethod
     def unknown(what: str) -> None:
         print(f"PANIC! unknown output format {what}")
 
 
+class ContainingEnum(StrEnum):
+    """
+    A string enum where one value may contain any number of other values from that same enum.
+    Similar to Flag and IntFlag, but for strings.
+    Create a subclass and define a dict `__contains` that maps each enum value to enum values it is supposed to contain.
+    This allows to a test like:
+         YourEnum.item1 in YourEnum.item2
+         True iff `item2` is defined to contain `item1` through `__contains`
+    As a convenience als automatically converts strings to enum instances for easy testing:
+        "foo" in YourEnum.item2
+        True iff "foo" converts to a `YourEnum` instance and is contained in `YourEnum.item2`
+    NOTE You will need to explicitly define whether an enum item is supposed to contain itself.
+    """
+
+    @property
+    def __contains(self) -> dict[ContainingEnum, set[ContainingEnum, ...]]:
+        """
+        dynamical name mangling to make override in subclass effective
+        """
+        return getattr(self, f"_{self.__class__.__name__}__contains")
+
+    def __contains__(self, other: Any) -> bool:
```

Comment:
This saves an import:

```suggestion
    def __contains__(self, other: object) -> bool:
```

Reply:
It is less explicit and the import is part of a multi-item import of which the other items remain needed.